src/store/store.js

```diff
@@ -0,0 +1,11 @@
+import { createStore, combineReducers, applyMiddleware } from "redux";
+import { composeWithDevTools } from "redux-devtools-extension";
+import thunk from "redux-thunk";
+import reducer from './reducer';
+const reducers = combineReducers({
+  reducer: reducer
```

Comment:
I mentioned the name above, and I'll reinforce it here. Instead of...

```js
import reducer from './reducer';
const reducers = combineReducers({
  reducer: reducer
});
```

... try naming things more deliberately, e.g.

```js
import response from './response-reducer';
const reducers = combineReducers({
  response
});
```